Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Collecting rackspace-auth-openstack==1.3
  Downloading http://localhost/packages/source/r/rackspace-auth-openstack/rackspace-auth-openstack-1.3.tar.gz
setuptools must be installed to install from a source distribution
Package search: {'_pypi_ordering': False, 'version': '1.3', 'name': 'rackspace-auth-openstack', 'summary': 'Rackspace Auth Plugin for OpenStack Clients.'}
Creating standard recipe for rackspace-auth-openstack-1.3
Using url https://pypi.python.org/packages/source/r/rackspace-auth-openstack/rackspace-auth-openstack-1.3.tar.gz (7 KB) for rackspace-auth-openstack.
Downloading rackspace-auth-openstack
Using cached download
Unpacking rackspace-auth-openstack...
done
working in /tmp/tmpuWHqICconda_skeleton_rackspace-auth-openstack-1.3.tar.gz
Fetching package metadata: ......

WARNING: conda-build appears to be out of date. You have version 1.13.0 but the
latest version is 1.14.0. Run

conda update -n root conda-build

to get the latest version.

Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
Traceback (most recent call last):
  File "setup.py", line 16, in <module>
    import setuptools
ImportError: No module named setuptools

Applying patch: '/tmp/tmpuWHqICconda_skeleton_rackspace-auth-openstack-1.3.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmpuWHqICconda_skeleton_rackspace-auth-openstack-1.3.tar.gz/rackspace-auth-openstack-1.3
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install rackspace-auth-openstack==1.3
Error: Could not pip install rackspace-auth-openstack==1.3
